### DISCUSSION TITLE
What cpu doing while offloading all layers in GPU

### DISCUSSION THREAD

**Discussion by FNsi** [ggml-org/llama.cpp#2215] [Q&A]
We already know, with a dumb gpu, the speed is lower than the CPU mode, even -t let the cpu usage 100%.

That's the real question. 

(that might be changed cause I didn't update llama.cop a week, but I did not seen any related in recent commits.)

**Comment by bullno1** [comment]
Last time I checked, spin waiting.

It's a known issue.

It makes sense for CPU or hybrid CPU&GPU though.

**Reply by FNsi** [reply]
I am thinking might it be possible like mmap both vram and ram utilise them...
